web/src/routes/(user)/partners/[userId]/+page.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	import type { PageData } from './$types';
+	import AssetGrid from '$lib/components/photos-page/asset-grid.svelte';
+	import UserPageLayout from '$lib/components/layouts/user-page-layout.svelte';
+
+	export let data: PageData;
+</script>
+
+<UserPageLayout user={data.user} title="{data.partner.firstName} {data.partner.lastName}'s Photos">
+	<AssetGrid user={data.partner} />
```

Comment:
The thumbnails in the asset grid can be selected, but the action bar doesn't show up and the scrubbable scollbar isn't present like in the main timeline. Would be good to keep the experience the same regardless if users are viewing their own timeline or that of others.

Reply:
That might be a bit complicated. What actions would need to be enabled for the selected assets? Do we support adding partner assets to albums, etc.? Might be non trivial to have that correctly integrated. It seems like there will be some nuances with it. I'd be happy with read only for now and another PR for actions later.

Reply:
I see Google Photos allows a limited set of actions for partner's photos (view, info and save to the user's library only). At the moment actions are only available while viewing assets in `AssetViewer` (copy, download, info)

I haven't noticed the scrollbar issue myself while testing. I don't like that this page doesn't have the same scroller as the main timeline, but looking into the code, I unfortunately can't understand why. I could add `slot="content"` attribute to the `AssetGrid` to fix it, but in this case the page loses its header

Reply:
I've changed page layout completely and the scrollbar is present and works as expected now